docker/test-images/zipkin-eureka/src/main/resources/application.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2015-2024 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# Add configuration to disable as much server caching as possible. Note that
+# not all configuration here are defined by netflix/eureka, rather some are
+# specific to spring-cloud/spring-cloud-netflix.
+eureka:
+  client:
```

Comment:
last couple hours died due to finding these settings. basically bad luck in CI where the first test passed into a stale data interval which outlasted awaitility's threshold!